README.md

```diff
@@ -20,34 +20,34 @@ TBD
 
 ## 🚛 Roadmap
 
-| Task | Complete? |
-| --- | --- |
-| Setup repo | ✅ |
-| README | ✅ |
-| Intial Unit/Request tests | |
-| Endpoint placeholders | |
-| Pydantic models | |
-| Update tests | |
-| DB Mocks | |
-| Query validation | |
-| Path validation | |
-| Sentry integration | |
-| Authentication | |
-| Rate-limiting | |
+| Task                      | Complete? |
+| ------------------------- | --------- |
+| Setup repo                | ✅        |
+| README                    | ✅        |
+| Intial Unit/Request tests |           |
+| Endpoint placeholders     |           |
+| Pydantic models           |           |
+| Update tests              |           |
+| DB Mocks                  |           |
+| Query validation          |           |
+| Path validation           |           |
+| Sentry integration        |           |
+| Authentication            |           |
+| Rate-limiting             |           |
 
 ## 🤝 Contributing
 
 For information on contributing to this project, please see the [contributing guide](CONTRIBUTING.md) and our [code of conduct](CODE_OF_CONDUCT.md).
 
 ## 🔗 Related
 
-* [Open Brewery DB Website & Documentation](https://github.com/chrisjm/openbrewerydb-gatsby)
-* [Open Brewery DB Dataset](https://github.com/openbrewerydb/openbrewerydb)
+- [Open Brewery DB Website & Documentation](https://github.com/chrisjm/openbrewerydb-gatsby)
```

Comment:
This should be updated to `[Open Brewery DB Website & Documentation](https://github.com/openbrewerydb/openbrewerydb-sveltekit)`